components/BridgeContent/SocketTxHistory/utils.tsx

```diff
@@ -0,0 +1,46 @@
+import { ActiveRoute, Status, StatusData } from 'types/Bridge';
+import SuccessIcon from '@mui/icons-material/CheckCircleRounded';
+import ErrorIcon from '@mui/icons-material/ErrorRounded';
+import InfoIcon from '@mui/icons-material/InfoRounded';
+import i18n from 'i18n';
+import chains from '../chains.json';
+
+export function routeToTxData(route: ActiveRoute) {
+  const { userTxs, bridgeTxHash, routeStatus, fromChainId } = route;
+  const isBridge = userTxs.some(({ steps }) => steps?.some(({ type }) => type === 'bridge'));
+  const isSwap = userTxs.some(({ steps }) => steps?.some(({ type }) => type === 'swap'));
+
+  const status: Record<Status, StatusData> = {
+    READY: {
+      statusLabel: i18n.t('Ready'),
```

Comment:
use `t` from `useTranslations`, this will not be translated if the language changes on the fly. Same goes for the ones below